crates/goose/src/scheduler.rs

```diff
@@ -1260,10 +737,9 @@ async fn run_scheduled_job_internal(
         .apply()
         .await
     {
-        tracing::error!("[Job {}] Failed to update session metadata: {}", job.id, e);
+        tracing::error!("Failed to update session: {}", e);
     }
 
-    tracing::info!("Finished job: {}", job.id);
```

Comment:
I know we're all about reducing the logging but it does seem like logging the start/stop of each job execution would not be a bad idea.

Or is it that we log it outside of this function?